subsys/bluetooth/host/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ zephyr_library_link_libraries(subsys__bluetooth)
 
 zephyr_library_sources_ifdef(CONFIG_BT_HCI_RAW          hci_raw.c hci_common.c)
 zephyr_library_sources_ifdef(CONFIG_BT_DEBUG_MONITOR    monitor.c)
+zephyr_library_sources_ifdef(CONFIG_BT_DEBUG_MONITOR_RTT monitor.c)
```

Comment:
Specifying `monitor.c` in two places looks a big odd, is that common practice? Perhaps it would make sense to have a high-level virtual option that both the UART- and -RTT-based options select, and then include `monitor.c` based on that virtual option (e.g. `CONFIG_BT_MONITOR`). Is the original `CONFIG_BT_DEBUG_MONITOR` strictly for UART usage now? If so, perhaps better to append `_UART` to it to make this clear?